<launch>
	
	<node pkg="perception3d" type="data_publisher_joint_test" name="data_publisher_joint_test" respawn="false" output="screen" >
	
		<param name="joint_name" type="int" value="1"/>
		<param name="joint_velocity_number" type="int" value="4" />
		<param name="joint_velocity0" type="int" value="10"/>
		<param name="joint_velocity1" type="int" value="5"/>
		<param name="joint_velocity2" type="int" value="1"/>
		<param name="joint_velocity3" type="int" value="2"/>
	</node>
	
</launch>
	
